src/content/changes/highlights/2022-09-22.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2022-09-22T10:00:00
+title: Highlights for the week ending September 23, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## User interfaces
+
+As of the latest version, our web UI is ready for use on Google Cloud Platform (GCP) installations.
+
+kubectl-gs [v2.23.0](https://github.com/giantswarm/kubectl-gs/releases/tag/v2.23.0) adds the capability to create a client certificate for workload cluster access with a specific CN prefix. This means you can bind the client presenting this certificate to an RBAC role as a user. Check the `login` command's new `--cn-prefix` flag.
+
```

Comment:
```suggestion

## Apps

- [security-pack-helper](https://github.com/giantswarm/security-pack-helper) version [v0.0.1](https://github.com/giantswarm/security-pack-helper/blob/main/CHANGELOG.md#001---2022-09-20) initial release supports watching and deleting Kyverno `ReportChangeRequest` resources when they exceed a configured threshold.
- [trivy-app](https://github.com/giantswarm/trivy-app) version [v0.6.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#060---2022-09-16) updates to Trivy version `0.30.4`.

```

Reply:
Sorry, but the announcement is already sent. We'll have to announce this with next week's round.